src/conntrack/nlas/nla.rs

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: MIT
+
+use derive_more::{From, IsVariant};
```

Comment:
I plan to remove the use of `derive_more`. Please do not use it anymore in new code.